css/dashboard.css

```diff
@@ -146,3 +178,161 @@ main .card {
   padding: 1rem;
   margin-right: 0.5rem;
 }
+
+tbody tr:not(:first-child) {
+  border-top: 1px solid #eaedf3;
+}
+
+tr th {
+  font-size: 14px;
+  line-height: 16px;
+
+  /* Black */
+  color: #222222;
+}
+
+tr td {
+  padding: 1.5rem 0.25rem !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5469161)